django/db/backends/sqlite3/features.py

```diff
@@ -28,6 +28,8 @@ class DatabaseFeatures(BaseDatabaseFeatures):
     has_case_insensitive_like = True
     # Is "ALTER TABLE ... RENAME COLUMN" supported?
     can_alter_table_rename_column = Database.sqlite_version_info >= (3, 25, 0)
+    # is "ALTER TABLE ... DROP COLUMN" supported?
+    can_alter_table_drop_column = Database.sqlite_version_info >= (3, 35, 0)
```

Comment:
Ideally we can pin at `3.35` as you've done here, but perhaps we should pin at `3.35.5` because `3.35.11` included bugfixes for that new functionality, and more importantly `3.35.5` has the following fix:
> Fix defects in the new ALTER TABLE DROP COLUMN feature that could corrupt the database file

Though it unfortunately doesn't specify further, or linked to a ticket/discussion. ([Docs for the entire 3.35 release line, including patch releases](https://www.sqlite.org/releaselog/3_35_5.html))

Reply:
I'm not sure. We usually use the version when the feature was introduced :thinking: For example `can_return_columns_from_insert` is pinned to `3.35` but some issues with `RETURNING` were fixed in `3.35.4`.

Reply:
Yeah I thought that might be the case, but thought it worth pointing out due to the sheer destructiveness of this specific fix in `.5`
FWIW it doesn't look _entirely_ without precedent to use the patch releases (though it is _rare_), insofar as `supports_aggregate_filter_clause` is pinned to a minimum of `3.30.1` where the actual functionality was added in `3.30.0` but had fixes in the `.1` release by my reading of the release notes.